src/en/charms-resources.md

```diff
@@ -0,0 +1,103 @@
+Title: Resources
+TODO:  Critical: Improve examples
+
+# Juju Resources
+
+A Juju *resource* is additional content/files that a charm needs in order to
```

Comment:
it doesn't *always* need them - they may be optional. Perhaps "that a charm can make use of or may require"

Reply:
OK